release/models/vlan/openconfig-vlan.yang

```diff
@@ -117,7 +117,28 @@ module openconfig-vlan {
   grouping vlan-state {
```

Comment:
Given that these leaves are in the `config` container above, it seems like we shouldn't need to redefine them? I see them in both containers today:

```
        +--rw vlans
        |  +--rw vlan* [vlan-id]
        |     +--rw vlan-id    -> ../config/vlan-id
        |     +--rw config
        |     |  +--rw vlan-id?   oc-vlan-types:vlan-id
        |     |  +--rw name?      string
        |     |  +--rw status?    enumeration
        |     +--ro state
        |     |  +--ro vlan-id?   oc-vlan-types:vlan-id
        |     |  +--ro name?      string
        |     |  +--ro status?    enumeration
```

Was there some other reason to redefine them?

Reply:
Ok